components/script/dom/bindings/utils.rs

```diff
@@ -7,13 +7,19 @@
 use crate::dom::bindings::codegen::InterfaceObjectMap;
 use crate::dom::bindings::codegen::PrototypeList;
 use crate::dom::bindings::codegen::PrototypeList::{MAX_PROTO_CHAIN_LENGTH, PROTO_OR_IFACE_LENGTH};
-use crate::dom::bindings::conversions::{jsstring_to_str, private_from_proto_check};
-use crate::dom::bindings::error::throw_invalid_this;
-use crate::dom::bindings::inheritance::TopTypeId;
+use crate::dom::bindings::conversions::{jsstring_to_str, private_from_proto_check, DerivedFrom};
+use crate::dom::bindings::error::Error;
+use crate::dom::bindings::error::{throw_dom_exception, throw_invalid_this};
+use crate::dom::bindings::htmlconstructor::html_constructor;
```

Comment:
Given that components/script/dom/bindings/htmlconstructor.rs exists (I forgot about it), let's move this code into that file.